Executive.js

```diff
@@ -76,4 +81,27 @@ class Executive {
 			new Arrow(1, 2, 3, 2, true),  // PHSX 210 to PHSX 216
 		]);
 	}
+}
+
+window.onload = () => {
+	var test = new Executive();
```

Comment:
The Executive object is already created when the page loads (the only code in index.html). The code you added here could be part of the Executive constructor instead of sitting outside a class.